### PR TITLE
Replace `GlobalTransform` with `Transform` in all examples

### LINKED ISSUE #1053: Use the Transform component for all examples
The ability to only use the `Transform` component without `GlobalTransform` was introduced recently and to reduce confusion for new users, I think it would be a good idea to make all examples and the book refer only to `Transform` instead of a mixture between the two.

### PR DESCRIPTION
**Pull request by moflipeje**
Hey, I'm new here and decided to tackle an easy issue. A few unrelated files got caught up in my `cargo fmt` pass—I hope that's fine. This should resolve #1053.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,12 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 * `amethyst_test` test framework for ergonomic testing of Amethyst applications ([#1000])
 
+### Changed
+
+* Replaced `GlobalTransform` with `Transform` in all examples. ([#1070])
```
Comment (Moxinilian) [inline docs/CHANGELOG.md]: You probably don't need to mention that in the changelog because it's only a meta change.

Comment (moflipeje) [reply docs/CHANGELOG.md]: Got it. I wasn't exactly sure how big the changelog's scope was.

Comment (Moxinilian) [reply docs/CHANGELOG.md]: Basically you need to add something to the changelog when you change a visible behavior (new features, bug fixes, API changes, etc...) in the engine itself. Here it's just the examples so it's fine.

**Comment by Moxinilian** [comment]
Great! Could you also squash your commits so that the commit history looks nice?

**Comment by moflipeje** [comment]
Done.

File: book/src/prefab.md [hunk 2]
```diff
@@ -89,18 +89,17 @@ impl<'a> PrefabData<'a> for Transform {
...
 
-Next we defined the `add_to_entity` function, which is used to actually instantiate data. In our case here,
-we insert a default `GlobalTransform` and the local `Transform` data on the referenced `Entity`. In this
-scenario we aren't using the third parameter to the function. This parameter contains a list of all `Entity`s
-affected by the `Prefab`, the first entry in the list will be the main `Entity`, and the rest will be the 
-`Entity`s that were created for all the entries in the data list inside the `Prefab`.
+Next, we define the `add_to_entity` function, which is used to actually instantiate data. In our case here,
+we insert the local `Transform` data on the referenced `Entity`. In this scenario we aren't using the third
+parameter to the function. This parameter contains a list of all `Entity`s affected by the `Prefab`, the first
+entry in the list will be the main `Entity`, and the rest will be the `Entity`s that were created for all the
```
[the remark this answers is not in the document]
Comment (torkleyy) [reply book/src/prefab.md]: Why?

**Comment by torkleyy** [comment]
24 hours have passed, let's merge this PR. Thanks for your first contribution, @moflipeje!

bors r+

**Pull request merged**